src/main/java/adris/altoclef/Settings.java

```diff
@@ -189,104 +176,75 @@
-     *
-     * WARNING: The butler will only accept non-chat messages as commands, but don't make this too lenient,
-     *      else you may risk unauthorized control to the bot. Basically, make sure that only whispers can
-     *      create the following messages.
+     * <p>
+     * {from}: Who the message was sent from {to}: Who the message was sent to, butler will ignore if this is not your username. {message}:
+     * The message.
+     * <p>
+     * <p>
+     * WARNING: The butler will only accept non-chat messages as commands, but don't make this too lenient, else you may risk unauthorized
+     * control to the bot. Basically, make sure that only whispers can create the following messages.
      */
-    private String[] whisperFormats = new String[] {
+    private String[] whisperFormats = {
+            "\\[{from} -> {to}\\] {message}",
             "{from} whispers to you: {message}",
-            "{from} whispers: {message}",
-            "\\[{from} -> {to}\\] {message}"
-    };
+            "{from} whispers: {message}"
+    }; // TODO: 2021-05-22 a regex would be better
 
     /**
-     * If true, the bot will perform basic survival tasks when no commands are in progress
-     * (eat food, force field mobs, etc.)
-     * It will only perform survival tasks allowed by other parameters in the settings file.
+     * If true, the bot will perform basic survival tasks when no commands are in progress (eat food, force field mobs, etc.) It will only
+     * perform survival tasks allowed by other parameters in the settings file.
      */
-    private boolean idleWhenNotActive = false;
+    private boolean idleWhenNotActive;
 
     /**
      * If we need to throw away something, throw away these items first.
      */
-    private int[] throwawayItems = new int[] {
+    private int[] throwawayItems = {
             // Overworld junk
-            Item.getRawId(Items.DIORITE),
-            Item.getRawId(Items.ANDESITE),
-            Item.getRawId(Items.GRANITE),
-            Item.getRawId(Items.COBBLESTONE),
-            Item.getRawId(Items.DIRT),
-            Item.getRawId(Items.GRAVEL),
+            Item.getRawId(Items.DIORITE), Item.getRawId(Items.ANDESITE), Item.getRawId(Items.GRANITE), Item.getRawId(Items.COBBLESTONE),
+            Item.getRawId(Items.DIRT), Item.getRawId(Items.GRAVEL),
             // Nether junk, to be fair it's mostly tuned for the "beat game" task
-            Item.getRawId(Items.NETHERRACK),
-            Item.getRawId(Items.MAGMA_BLOCK),
-            Item.getRawId(Items.SOUL_SOIL),
-            Item.getRawId(Items.SOUL_SAND),
-            Item.getRawId(Items.NETHER_BRICKS),
-            Item.getRawId(Items.NETHER_BRICK)
-    };
+            Item.getRawId(Items.NETHERRACK), Item.getRawId(Items.MAGMA_BLOCK), Item.getRawId(Items.SOUL_SOIL), Item.getRawId(
+            Items.SOUL_SAND), Item.getRawId(Items.NETHER_BRICKS), Item.getRawId(Items.NETHER_BRICK)
+    }; // TODO: 2021-05-22 please stop storing raw ids lol
```

Comment:
dang was just thinking about this, yeah you're 100% right. Raw IDs are ugly as hell. I'm thinking storing strings/translation keys instead or something.